Automattic/wp-calypso
# Translatable String: fix error in Calypso

## Proposed Changes

* Change to ternary to prevent errors in console

## Why are these changes being made?

Currently in Calypso there is an error that says: `Invalid value for Translatable string in SidebarItem`. This is coming from a type check for translate strings https://github.com/Automattic/wp-calypso/blob/trunk/client/components/translatable/proptype.js

For tooltips `showTooltip && title` returns as `false` which is not a translate callback as expected. Changing this to be undefined if `showTooltip` is not enabled.

## Testing Instructions

* Using live link go to My Sites and check to make sure there is no regressions.
